src/tools/sssctl/sssctl_data.c

```diff
@@ -290,3 +291,61 @@ errno_t sssctl_cache_upgrade(struct sss_cmdline *cmdline,
 
     return EOK;
 }
+
+errno_t sssctl_cache_expire(struct sss_cmdline *cmdline,
+                            struct sss_tool_ctx *tool_ctx,
+                            void *pvt)
+{
+    errno_t ret;
+    char *cmd_args = NULL;
+    const char *cachecmd = SSS_CACHE;
+    char *cmd = NULL;
+    int i;
+
+    if (cmdline->argc == 0) {
+        fprintf(stderr, _("Missing option. \n"));
+        cachecmd = "sss_cache --help";
```

Comment:
Please do not append help here.
Help is already displayed in case of missing  arguments
```
[root@graviton ~]# sss_cache
Usage: sss_cache [-?EUGNSAHR] [-?|--help] [--usage] [-E|--everything]
        [-u|--user=STRING] [-U|--users] [-g|--group=STRING] [-G|--groups]
        [-n|--netgroup=STRING] [-N|--netgroups] [-s|--service=STRING]
        [-S|--services] [-a|--autofs-map=STRING] [-A|--autofs-maps]
        [-h|--ssh-host=STRING] [-H|--ssh-hosts] [-r|--sudo-rule=STRING]
        [-R|--sudo-rules] [-d|--domain=STRING]
Please select at least one object to invalidate
``` 

And it will make patch simpler :-)